### PR TITLE
Remove isPublic from Entry entity fix #2598

### LINKED ISSUE #2598: Remove isPublic attribute for Entry
We can now share entry publicly. The `isPublic` attribute in Entry class is not useful.

### PR DESCRIPTION
**Pull request by bourvill**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Documentation | not needed
| Translation   | no
| Fixed tickets | #2598 
| License       | MIT

Refer to #2598

### CODE REVIEW

File: admin-export.json [hunk 1]
```diff
@@ -0,0 +1 @@
...ext":"This is my 2nd annotation \/o\/","quote":"content"}],"mimetype":"text\/html","language":"fr","reading_time":1,"domain_name":"domain.io"},{"is_archived":0,"is_starred":0,"tags":["foo bar","bar"],"id":4,"title":"test title entry4","url":"http:\/\/0.0.0.0\/entry4","content":"This is my content \/o\/","created_at":"2017-04-13T12:51:10+0200","updated_at":"2017-04-13T12:51:10+0200","annotations":[],"mimetype":"text\/html","language":"en","reading_time":12,"domain_name":"domain.io"},{"is_archived":0,"is_starred":1,"tags":[],"id":5,"title":"test title entry5","url":"http:\/\/0.0.0.0\/entry5","content":"This is my content \/o\/","created_at":"2017-04-13T12:51:10+0200","updated_at":"2017-04-13T12:51:10+0200","annotations":[],"mimetype":"text\/html","language":"fr","reading_time":12,"domain_name":"domain.io","preview_picture":"http:\/\/0.0.0.0\/image.jpg"},{"is_archived":1,"is_starred":0,"tags":["bar"],"id":6,"title":"test title entry6++","url":"http:\/\/0.0.0.0\/entry6","content":"This is my content \/o\/","created_at":"2017-04-13T12:51:10+0200","updated_at":"2017-04-13T12:52:07+0200","annotations":[],"mimetype":"text\/html","language":"de","reading_time":12,"domain_name":"domain.io"}]
```
Comment (j0k3r) [inline admin-export.json]: :thinking:

File: specialexport.json [hunk 2]
```diff
@@ -0,0 +1 @@
...ext":"This is my 2nd annotation \/o\/","quote":"content"}],"mimetype":"text\/html","language":"fr","reading_time":1,"domain_name":"domain.io"},{"is_archived":0,"is_starred":0,"tags":["foo bar","bar"],"id":4,"title":"test title entry4","url":"http:\/\/0.0.0.0\/entry4","content":"This is my content \/o\/","created_at":"2017-04-13T12:51:10+0200","updated_at":"2017-04-13T12:51:10+0200","annotations":[],"mimetype":"text\/html","language":"en","reading_time":12,"domain_name":"domain.io"},{"is_archived":0,"is_starred":1,"tags":[],"id":5,"title":"test title entry5","url":"http:\/\/0.0.0.0\/entry5","content":"This is my content \/o\/","created_at":"2017-04-13T12:51:10+0200","updated_at":"2017-04-13T12:51:10+0200","annotations":[],"mimetype":"text\/html","language":"fr","reading_time":12,"domain_name":"domain.io","preview_picture":"http:\/\/0.0.0.0\/image.jpg"},{"is_archived":1,"is_starred":0,"tags":["bar"],"id":6,"title":"test title entry6++","url":"http:\/\/0.0.0.0\/entry6","content":"This is my content \/o\/","created_at":"2017-04-13T12:51:10+0200","updated_at":"2017-04-13T12:52:07+0200","annotations":[],"mimetype":"text\/html","language":"de","reading_time":12,"domain_name":"domain.io"}]
```
Comment (j0k3r) [inline specialexport.json]: :thinking: ²

File: admin-export.json (hunk 1 above)
Comment (tcitworld) [inline admin-export.json]: It's the generated json from console tests which outputs in the main directory. We should add them to `.gitignore`.

File: specialexport.json (hunk 2 above)
Comment (nicosomb) [inline specialexport.json]: argh, sorry, I did a git add -A ...

**Pull request merged**